src/btwa_api/scripts.py

```diff
@@ -10,3 +13,19 @@ def add_handler(username, password):
     session.add(handler)
     session.commit()
     session.close()
+
+
+@click.command("add-handler")
```

Comment:
What/who does the "handler" here refer to, exactly? Can't the name/term be more descriptive or accurate?

Reply:
Admin/epidemiologist. I know, the name sucks, but as I stated above, it will probably go away. If not I'll rename it.